code/game/machinery/computer/atmos_control.dm

```diff
@@ -203,48 +212,69 @@ GLOBAL_LIST_EMPTY(atmos_air_controllers)
 	sensors = list(ATMOS_GAS_MONITOR_SENSOR_O2 = "Oxygen Tank")
 	circuit = /obj/item/circuitboard/computer/atmos_control/tank/oxygen_tank
 
+/obj/machinery/computer/atmos_control/tank/oxygen_tank/offstation
+	can_reconnect = FALSE
+
 /obj/machinery/computer/atmos_control/tank/toxin_tank
 	name = "Plasma Supply Control"
 	input_tag = ATMOS_GAS_MONITOR_INPUT_TOX

```

Comment:
Aren't these all still connected to the station?

Reply:
They seem to connect correctly on mapload, probably because lavaland initializes early. Let me actually double check the station ones to make sure those are okay.

I did consider a z check on connecting to sensors, still might for safety.

Reply:
Yeah no my testing was bad. I'll need to figure out something else then.

Reply:
consoles gone